.github/workflows/helm-release-chart.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Helm Chart
+on:
+  push:
+    tags:
+     - v*
+    paths:
+    - chart/**
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v4
+
+    - name: Log in to GHCR
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Update Helm Dependencies
+      working-directory: chart
+      run: |
+        helm dependency update
+
+    - name: Package Helm Chart
+      working-directory: chart
+      run: |
+        helm package .
+        mv *.tgz ../packaged-chart.tgz
+
+    - name: Push Helm Chart to GHCR
+      env:
+        HELM_EXPERIMENTAL_OCI: 1
+      run: |
+        helm push packaged-chart.tgz oci://ghcr.io/${{ github.repository_owner }}/helm-charts
```

Comment:
shouldn't we name this `kyoo-helm` or something like that?

Reply:
It appends the package name, so it would be `oci://ghcr.io/zoriya/<path>/kyoo`, it's based on the name in [Chart.yaml](https://github.com/zoriya/Kyoo/blob/master/chart/Chart.yaml#L2).